### PR TITLE
Binary class support for AccuracyCallback

### PR DESCRIPTION
**Pull request by Dipet**

### CODE REVIEW

File: catalyst/dl/utils/criterion/accuracy.py [hunk 1]
```diff
@@ -1,15 +1,29 @@
 import numpy as np
+from catalyst.utils import get_activation_fn
 
 
-def accuracy(outputs, targets, topk=(1, )):
+def accuracy(outputs,
```
Comment (TezRomacH) [inline catalyst/dl/utils/criterion/accuracy.py]: code style, please

File: catalyst/dl/callbacks/metrics/accuracy.py [hunk 2]
```diff
@@ -44,6 +44,8 @@ def __init__(
         prefix: str = "accuracy",
         accuracy_args: List[int] = None,
         num_classes: int = None,
+        threshold: float = None,
+        activation: str = "Sigmoid"
```
Comment (TezRomacH) [inline catalyst/dl/callbacks/metrics/accuracy.py]: add new params to the documentation, please

**Review by TezRomacH — CHANGES_REQUESTED** [review]

**Comment by Scitator** [comment]
Please, uncheck the `WIP` label after your will resolve all conversations :)

File: catalyst/dl/utils/criterion/accuracy.py (hunk 1 above)
Comment (Dipet) [reply catalyst/dl/utils/criterion/accuracy.py]: @TezRomacH What code style is brocken here? `make codestyle` shows only docstrings warnings. On this page I cannot see specifics rules https://github.com/catalyst-team/catalyst/blob/master/CONTRIBUTING.md

Comment (Scitator) [reply catalyst/dl/utils/criterion/accuracy.py]: @Dipet catalyst codestyle
```
def accuracy(
    outputs,
    targets,
    topk=(1, ),
    threshold: float = None,
    activation: str = None
):
```
This format allows your to refactor all your function names more easily.

**Comment by Dipet** [comment]
@Scitator how a I can uncheck WIP if I have no write access?

**Pull request merged**